pegen/c_generator.py

```diff
@@ -22,11 +22,22 @@
 
     if (!PyArg_ParseTuple(args, "s", &filename))
         return NULL;
-    return run_parser(filename, (void *)start_rule, %(mode)s);
+    return run_parser_from_file(filename, (void *)start_rule, %(mode)s);
+}
+
+static PyObject *
+parse_string(PyObject *self, PyObject *args)
+{
+    const char *the_string;
+
+    if (!PyArg_ParseTuple(args, "s", &the_string))
+        return NULL;
+    return run_parser_from_string(the_string, (void *)start_rule, %(mode)s);
 }
 
 static PyMethodDef ParseMethods[] = {
-    {"parse",  parse_file, METH_VARARGS, "Parse a file."},
+    {"parse_file",  parse_file, METH_VARARGS, "Parse a file."},
+    {"parse_string",  parse_string, METH_VARARGS, "Parse a string."},
```

Comment:
If you're changing the public name of the function, you should also change the lines where the function is called in the `Makefile`.

Reply:
Yep, I realized afterwards in the Travis-CI failure thanks to #13 (That payed off very quickly) :D